travis-run-tests.sh

```diff
@@ -24,9 +24,7 @@ fi
 if [ $1 == "group4" ]; then
   cd app-authz-springboot && mvn -B -s ../maven-settings.xml clean test -Pspring-boot -q
   cd ../service-springboot-rest && mvn -B -s ../maven-settings.xml clean test -Pspring-boot -q
-  mvn spring-boot:run >/dev/null&
-  cd ../app-springboot
-  mvn -B -s ../maven-settings.xml clean test -Pspring-boot
+  cd ../app-springboot && mvn -B -s ../maven-settings.xml clean test -Pspring-boot
```

Comment:
Why is "mvn spring-boot:run >/dev/null&" being removed? Got no clue why it was there before, but I assume there was some reason.

Reply:
I modified the Spring Quickstart to work with CI. I *think* it never did (someone obviously was trying to implement it but didn't finish the work). With my changes - this quickstarts worked fine.